src/chain/blob.ts

```diff
@@ -0,0 +1,58 @@
+import {ABISerializableObject} from '../serializer/serializable'
+import {ABIEncoder} from '../serializer/encoder'
+import {arrayEquals, arrayToHex, isInstanceOf} from '../utils'
+
+export type BlobType = Blob | string
+
+export class Blob implements ABISerializableObject {
+    static abiName = 'blob'
+
+    /**
+     * Create a new Bytes instance.
+     * @note Make sure to take a [[copy]] before mutating the bytes as the underlying source is not copied here.
+     */
+    static from(value: BlobType): Blob {
+        if (isInstanceOf(value, this)) {
+            return value
+        }
+        if (typeof value === 'string') {
+            return this.fromString(value)
+        }
+        throw new Error('Invalid blob')
+    }
+
+    static fromString(value: string) {
```

Comment:
This needs to handle the invalid base64 padding nodeos outputs

Reply:
I think I have tests ensuring the invalid stuff works here: https://github.com/wharfkit/antelope/pull/71/commits/1cd6b9b385b3ad7ec8ce67dff1aaea94bdb8c57f